sql-server-2014-alwayson-dsc/azuredeploy.json

```diff
@@ -150,12 +128,67 @@
       },
       "defaultValue": "sqlservice"
     },
+
     "sqlServerServiceAccountPassword": {
       "type": "securestring",
       "metadata": {
         "description": "The SQL Server Service account password"
       }
     },
+
+    "VmNetworkSubnet": {
```

Comment:
Can this and the addresses below become a variable and not ask the user to enter so many parameters?